packages/cva/src/index.ts

```diff
@@ -33,21 +33,36 @@ type ClassValue =
   | null
   | boolean
   | undefined;
-type ClassDictionary = Record<string, any>;
+type ClassDictionary = Record<string, unknown>;
```

Comment:
If possible I'd like to keep this as `any` to ensure compatibility with `clsx` 🙏 
https://github.com/lukeed/clsx/blob/master/clsx.d.ts#L3

Reply:
(Sorry for the delay, getting back to reviewing)

Reply:
Fine with me 👍, updated ✅

Reply:
Thank you!